Mono.Debugging/Mono.Debugging.Client/AssemblyEventArgs.cs

```diff
@@ -25,18 +25,24 @@
 // THE SOFTWARE.
 
 using System;
+using System.Collections.Generic;
 
 namespace Mono.Debugging.Client
 {
 	public class AssemblyEventArgs : EventArgs
 	{
-		public AssemblyEventArgs (string location)
+		public AssemblyEventArgs (string location, List<Assembly> assemblies)
 		{
 			Location = location;
+			Assemblies = assemblies;
 		}
 
 		public string Location {
 			get; private set;
 		}
+
+		public List<Assembly> Assemblies {
```

Comment:
Can we pass the `Assembly` in the event instead? That would also solve the problem of the `assemblies` list having unloaded assemblies in it.

Theoretically, any kind of UI should not need to get all these assemblies in the event arguments, just the latest change.